docs/faq.rst

```diff
@@ -1,6 +1,20 @@
 FAQ
 ===
 
+What happens if I rename my GitHub repository?
+----------------------------------------------
+
+If you have registered a tool or workflow from GitHub in Dockstore the
+link to the repository for your tool or workflow is displayed on the
+Info tab next to 'Source Code'. If you then `rename <https://docs.github.com/en/github/administering-a-repository/renaming-a-repository>`__
+the GitHub repository, the Source Code link will be incorrect, but
```

Comment:
I wouldn't say incorrect; I'd say it shows the old repository name.

Reply:
changed 'be incorrect' to 'display the original name'